File: pset6/mario/more/mario.py
"""
Prints brics from mario
height = 2 will give
 #  #
##  ##
"""


def main():
    while(True):
        try:
            height = int(input("Enter a positive number less than 23: "))
        except:
            continue
        else:
            if height <= 23 and height >= 0:
                break
    j = height - 1
    for i in range(height):
        print(" " * j, end="")# Repeat " " j times
        print("#" * (height - j), end="")# Repeat "#" height - j times
        print("  ", end="")# print space "  "
        print("#" * (height - j), end="")# Repeat "#" height - j times
        print()# for new line
        j -= 1


if __name__ == "__main__":
    main()

File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    //Get Positive Height less than 23 from the user
    do
    {
        height = get_int("Enter a positive number less than 23: ");

    }
    while (height > 23 || height < 0);
    // Loop for the Rows to be printed, j will be number of whitespaces
    for (int i = 0, j = (height - 1); i < height; i++, j--)
    {
        // Loop for whitespaces
        for (int k = 0; k < j; k++)
        {
            printf(" ");
        }
        // Loop for Brics
        for (int k = 0; k < (height + 1) - j; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}